### PR TITLE
Initial check-in of TIX clock app

### PR DESCRIPTION
**Pull request by pawliger**
Initial check-in of the TIX clock app for Tidbyt. Displays the times using colorful blocks. Includes 12/24 hour clock settings and color theme choices.

### CODE REVIEW

**Comment by pawliger** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by pawliger** [comment]
recheck

File: apps/tixclock/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+id: tix-clock
+name: TIX Clock
+summary: Coloful blocks tell time
+desc: A Tidbyt version of the TIX clock. It uses arrays of colorful blocks to show the time.
+author: pawliger, original TIX Clock by James Thompson
```
Comment (matslina) [inline apps/tixclock/manifest.yaml:6]:
```suggestion
author: pawliger
```

File: apps/tixclock/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+id: tix-clock
+name: TIX Clock
+summary: Coloful blocks tell time
+desc: A Tidbyt version of the TIX clock. It uses arrays of colorful blocks to show the time.
```
Comment (matslina) [inline apps/tixclock/manifest.yaml:5]:
```suggestion
desc: A Tidbyt version of the TIX clock. It uses arrays of colorful blocks to show the time. The original TIX Clock was invented by James Thompson.
```

**Review by matslina — CHANGES_REQUESTED** [review]
Thanks for the PR. Looks great! A few comments inline.

**Comment by pawliger** [comment]
Any thoughts when this might be reviewed? Thanks.

**Review by matslina — APPROVED** [review]

**Pull request merged**